src/System.Security.Cryptography.Encoding/tests/System.Security.Cryptography.Encoding.Tests.csproj

```diff
@@ -21,6 +21,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AsnEncodedData.cs" />
+    <Compile Include="AsnEncodedDataCollection.cs" />
```

Comment:
Port-from-TFS was the name of the class it was testing.  Ones I added later end in Tests to match the class name (and not have the class name conflict with that of what it's testing).

So this should probably be AsnEncodedDataCollectionTests.cs